src/libs/ReportUtils.ts

```diff
@@ -550,12 +550,12 @@ function isDraftExpenseReport(report: OnyxEntry<Report>): boolean {
 /**
  * Given a collection of reports returns them sorted by last read
  */
-function sortReportsByLastRead(reports: OnyxCollection<Report>): Array<OnyxEntry<Report>> {
+function sortReportsByLastRead(reports: OnyxCollection<Report>, reportsMetadata: OnyxCollection<ReportMetadata>): Array<OnyxEntry<Report>> {
```

Comment:
@DylanDylann Instead of passing reportsMetadata as a parameter, can we use global variable in ReportUtils and use that?

Reply:
Because the we are passing the reports as a parameter, so i think we should do it with reportsMetadata as well for consitency